crates/usvg/src/text/mod.rs

```diff
@@ -0,0 +1,29 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use crate::Text;
+
+mod layout;
+mod outline;
+
+pub use layout::{PositionedGlyph, Span, TextFragment};
```

Comment:
I would suggest exporting just the `layout` module. Just like we do with filters. To have a cleaner global namespace.

So the caller would use `layout::Span` instead of `Span`, which we already have in `Text`.